onecodex/analyses.py

```diff
@@ -303,10 +304,9 @@ def to_df(
         if normalize is False and self._guess_normalized():
             raise OneCodexException("Data has already been normalized and this can not be undone.")
 
-        if normalize is True or (
-            normalize == "auto" and rank is not None and self._field != "abundance"
-        ):
-            df = df.div(df.sum(axis=1), axis=0)
+        if normalize is True or (normalize == "auto" and rank):
+            if not self._guess_normalized():
+                df = df.div(df.sum(axis=1), axis=0)
```

Comment:
@boydgreenfield I figured out what's going on here. We've introduced `abundance_w_children` as a field for cumulative abundances, which opened a bug where you could double-normalize the abundance data. I moved the check for whether the data type is abundances into `_guess_normalized`.